editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -5673,6 +5674,84 @@ export var storyboard = (
         ])
       })
     })
+
+    describe('Pasting with steganography enabled', () => {
+      setFeatureForBrowserTestsUseInDescribeBlockOnly('Steganography', true)
+
+      it('steganography data is cleaned from replaced props', async () => {
+        const editor = await renderTestEditorWithCode(
+          `import * as React from 'react'
+        import { Storyboard, Scene } from 'utopia-api'
+        
+        const MyComponent = ({ title }) => {
+          return <div data-uid='root'>heeeello</div>
+        }
+        
+        const hello = 'hello'
+        
+        export var storyboard = (
+          <Storyboard data-uid='sb'>
+            <Scene
+            data-uid='scene'
+              style={{
+                backgroundColor: '#0091FFAA',
+                position: 'absolute',
+                left: 567,
+                top: 486,
+                width: 288,
+                height: 362,
+              }}
+            >
+              <MyComponent data-uid='component' title={hello} />
+            </Scene>
+          </Storyboard>
+        )
+        `,
+          'await-first-dom-report',
+        )
+
+        await selectComponentsForTest(editor, [EP.fromString('sb/scene/component')])
+        await pressKey('c', { modifiers: cmdModifier })
+        await runPaste(editor)
+
+        expect(getPrintedUiJsCode(editor.getEditorState())).toEqual(
+          formatTestProjectCode(`import * as React from 'react'
+        import { Storyboard, Scene } from 'utopia-api'
+
+        const MyComponent = ({ title }) => {
+          return <div data-uid='root'>heeeello</div>
+        }
+
+        const hello = 'hello'
+
+        export var storyboard = (
+          <Storyboard data-uid='sb'>
+            <Scene
+              data-uid='scene'
+              style={{
+                backgroundColor: '#0091FFAA',
+                position: 'absolute',
+                left: 567,
+                top: 486,
+                width: 288,
+                height: 362,
+              }}
+            >
+              <MyComponent data-uid='component' title={hello} />
+              <MyComponent
+                data-uid='com'
+                title='hello'
+                style={{ top: 0, left: 0, position: 'absolute' }}
+              />
+            </Scene>
+          </Storyboard>
+        )
+`),
+        )
+
+        expect(1).toEqual(1)
```

Comment:
```suggestion
```
I'm guessing this was to shut VS Code up whilst you were writing the test :)

Reply:
whoops, sorry, thanks for the suggestion!